src/Sql/dbo/Stored Procedures/User_UpdateStorage.sql

```diff
@@ -34,16 +35,30 @@ BEGIN
             #UserStorageUpdateTemp
     )
     SELECT
-        @Storage = SUM([CTE].[Size])
+        @AttachmentStorage = SUM([CTE].[Size])
     FROM
         [CTE]
 
     DROP TABLE #UserStorageUpdateTemp
 
+    ;WITH [CTE] AS (
+        SELECT
+            [Id],
+            CAST(JSON_VALUE([Data],'$.Size') AS BIGINT) [Size]
+        FROM
+            [Send]
+        WHERE
+            [UserId] = @Id
+    )
+    SELECT
+        @SendStorage = SUM([CTE].[Size])
+    FROM
+        [CTE]
+
     UPDATE
         [dbo].[User]
     SET
-        [Storage] = @Storage,
+        [Storage] = (@AttachmentStorage + @SendStorage),
```

Comment:
Either of these variables could be `null` if no records exists; I would imagine causing issues when performing this addition as the result would be `null` even if one of the two measures had a valid value.

![image](https://user-images.githubusercontent.com/3904944/97905774-9582ed80-1d10-11eb-994d-f633724bbd07.png)